.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# these owners will be requested for review when someone
+# opens a pull request.
+*       @dav3r @hillaryj @mcdonnnj
```

Comment:
These extra spaces should still be removed.

Reply:
Resolved in https://github.com/cisagov/assessment-data-import-terraform/pull/4/commits/4cc63bbe56ecd22a9e82a61dc847b4bf1c6a4d75